microservices/notification-tenant-lifecycle-consumer/dev/values.yaml

```diff
@@ -0,0 +1,26 @@
+name: interop-be-notification-tenant-lifecycle-consumer
+techStack: nodejs
+
+serviceAccount:
+  roleArn: "arn:aws:iam::505630707203:role/interop-be-notification-tenant-lifecycle-consumer-dev-es1"
+
+service:
+  create: false
+
+configmap:
+  KAFKA_GROUP_ID: "{{.Values.namespace}}-notification-tenant-lifecycle-consumer"
+  TOPIC_STARTING_OFFSET: "latest"
```

Comment:
This consumer and notification-user-lifecycle-consumer are set to `latest` assuming that we'll use another process to import previous data (which is what we're assuming especially for the selfcare consumer)